packages/evolution-common/package.json

```diff
@@ -20,6 +20,9 @@
     "format": "prettier-eslint $PWD/'src/**/*.{ts,tsx}' --write"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.32",
+    "@fortawesome/free-solid-svg-icons": "^5.15.1",
```

Comment:
Except for fontawesome-svg-core, do you need the other 2 packages? If not, please remove from dependencies